modules/plugins/flexboxIE.js

```diff
@@ -24,4 +33,41 @@ export default function flexboxIE(
   if (alternativeProps.hasOwnProperty(property)) {
     style[alternativeProps[property]] = alternativeValues[value] || value
   }
+  if (property === 'flex') {
+    // For certain values we can do straight mappings based on the spec
+    // for the expansions.
+    if (flexShorthandMappings.hasOwnProperty(value)) {
```

Comment:
This should probably use Object.prototype.hasOwnProperty.call, for safety

Reply:
Can certainly update it, was following the prior syntax already in place a few lines above at https://github.com/rofrischmann/inline-style-prefixer/pull/166/files#diff-1aebbd1a101132a004e54fd39eead4fbR33.